frontends/qt/main.cpp

```diff
@@ -124,6 +127,21 @@ class RequestInterceptor : public QWebEngineUrlRequestInterceptor {
             return;
         }
 
+        // All the requests originated in the wallet-connect section are allowed, as they are needed to
+        // load the Dapp logos and it is not easy to filter out non-images requests.
+        bool onWCPage = currentUrl.contains(QRegularExpression("^qrc:/account/[^\/]+/wallet-connect/.*$"));
+        if (onWCPage) {
+          info.block(false);
```

Comment:
are the ` info.block(false);` actually needed? not sure what they do.

Reply:
Right, they seem to be useless